indico/modules/rb_new/client/js/modules/roomList/filters/EquipmentForm.jsx

```diff
@@ -94,8 +100,28 @@ export default class EquipmentForm extends FilterFormComponent {
                                            this.setFeature(feat.name, checked);
                                        }} />
                     ))}
-                    {!!availableFeatures.length && !!availableEquipment.length && <br />}
-                    {availableEquipment.map(equip => (
+                    {!!availableFeatures.length && !!availableEquipment.length && (
+                        <>
+                            <br />
```

Comment:
Can't this be avoided with some CSS?

Reply:
maybe a margin on the accordion?

Reply:
Should I create a new module file just to add some css to the accordion?

Reply:
`EquipmentForm.module.scss`.. anyway, we had the `<br />` there before.. but using a margin instead would be nicer indeed

Reply:
`EquipmentForm.module.scss`.. anyway, we had the `<br />` there before.. but using a margin instead would be nicer indeed

Reply:
You even have `equipment-accordion`  styleName that's not being used.

Reply:
I think she already updated the PR but forgot to add the stylesheet ;)